deploy-board/deploy_board/templates/clusters/cluster_configuration.html

```diff
@@ -821,6 +823,11 @@ <h4 class="panel-title pull-left pointer-cursor">
                         return false
                     }
 
+                    if ('root_volume_type' in clusterInfo.configs && (clusterInfo.configs['root_volume_type'] != "gp2" && clusterInfo.configs['root_volume_type'] != "gp3")) {
```

Comment:
Can you do something similar to this in JS?
```suggestion
                    if ('root_volume_type' in clusterInfo.configs && clusterInfo.configs['root_volume_type']not in {"gp2", "gp3"})) {
```